tests/test_qb_timeline.py

```diff
@@ -420,3 +421,89 @@ def test_age(self):
         cache_key = QB_StatusCacheKey()
         cache_key.update(hourback)
         assert cache_key.minutes_old() == 60
+
+
+class HasAt(object):
+    """Mock the sorted portion of QBT for testing AtOrderedList"""
+    def __init__(self, at, label):
+        self.at = at
+        self.label = label
+
+
+n = HasAt(datetime.utcnow(), 'now')
```

Comment:
Please pick an arbitrary datetime, [non-deterministic tests can be flakey](https://martinfowler.com/articles/nonDeterminism.html#Time)